sw/device/lib/testing/test_framework/test_main.c

```diff
@@ -40,11 +40,11 @@ int main(int argc, char **argv) {
   bool result = test_main();
 
   // Must happen before any debug output.
-  if (kTestConfig.can_clobber_uart) {
+  if (kDeviceType != kDeviceSimDV && kTestConfig.can_clobber_uart) {
     init_uart();
+    test_coverage_send_buffer();
```

Comment:
Are we still intending to collect coverage on DIFs?

Reply:
Probably not, but some of the mask rom func tests use the test framework it looks like, and we probably want coverage info on the mask rom drivers.